common/test/acceptance/pages/studio/import_export.py

```diff
@@ -239,10 +239,16 @@ def is_task_list_showing(self):
 
     def is_timestamp_visible(self):
         """
-        Checks if the UTC timestamp of the last successfull import is visible
+        Checks if the UTC timestamp of the last successful import is visible
         """
         return self.q(css='.item-progresspoint-success-date').visible
 
+    def wait_for_timestamp_visible(self):
```

Comment:
Note to reviewers: this file and the next are a fix for a flaky test (which was failing reliably on our branch).